test/e2e/toolchaincluster_test.go

```diff
@@ -150,6 +144,31 @@ func verifyToolchainCluster(t *testing.T, await *wait.Awaitility, otherAwait *wa
 	})
 }
 
+func generateNewName(prefix, baseName string) string {
+	if len(prefix+baseName) >= 63 {
+		return prefix + baseName[len(prefix):]
```

Comment:
shouldn't this be something like 
```suggestion
		return prefix + baseName[len(prefix):63]
```

Reply:
I mean, removing the length of the prefix doesn't ensure that the name is < 63 chars right.

Reply:
ok, my assumption was that the name was already within the boundary

Reply:
I wanted to drop the prefix of the original name only, to keep the uniqueness of the names. But it's maybe not so important since we add some specific prefix anyway. Let me simplify it

Reply:
simplified b7c4c23e